doc/rst/developer/adding-a-tasking-model.rst

```diff
@@ -16,7 +16,7 @@ create and manage tasks, along with public types describing task
 identifiers and sync variable meta-information.  All of this needs
 defining when you add a tasking layer implementation.  The easiest way
 to proceed will be to copy an existing tasking model and modify it.  The
-``fifo`` tasking implementation is a good candidate, since it is the
+``fifo`` tasking implementation is a good candidate since it is the
```

Comment:
> I think the use of the comma here is fine, would you go into why you wanted to remove it?

I got this thing marked out by `grammarly` while reading the doc in my browser. Also please see [this](https://ell.stackexchange.com/questions/38051/should-i-add-a-comma-before-since)

Reply:
@lydia-duncan so what should we do?

Reply:
Sorry, I haven't had a chance to get back to this yet.  I'll try to take a look by the end of the day, but may end up needing to put it off until tomorrow

Reply:
Sure. No problem :-)

Reply:
So the link you've posted says "However, it's not *necessary* to put a comma before since when this dependent clause starts after the main clause." (emphasis mine).  That's different than saying it is wrong to do so - it makes it a matter of preference, and in cases of preference we should err on the side of not making changes